### DISCUSSION TITLE
Livewire component name

### DISCUSSION THREAD

**Discussion by AkramZerarka** [livewire/livewire#4919] [7. Bugs (archived)]
Hello everyone
I have a problem with name of components i think the problem in getName method
For example name of my component is Global.Backend.User.Company.Create i found in the fingerprint name lobal.backend.user.create the first letter (g) is deleted 
When i edit getName methode in component.php everything is work fine 
`(string) str($fullName)->substr(strlen($namespace));`

**Comment by joshhanley** [comment]
@AkramZerarka can you please share your code and any steps you followed so we can try and reproduce the issue? Thanks!
